cli/version/version.go

```diff
@@ -27,9 +27,14 @@ Usage of version:
 	cfssl version
 `
 
+// Returns the formatted version string.
```

Comment:
The Go standard is to have the function name in the docstring, so this should be 

```
// FormatVersion returns the formatted version string.
```

[golint](https://github.com/golang/lint) in our builds will catch this and cause the build to fail.